### PR TITLE
doc update to make things easier for the next new colleague

### PR DESCRIPTION
**Pull request by david-navapbc**
This PR ...

Updates the "getting-started" doc to include some more warnings about Macs with Apple CPUs, which credentials are necessary to log into the app when run locally, and step by step instructions for sending a sample message through the app. It also adds postman collections so we're no longer passing them around as tribal knowledge to folks who don't have them. 

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. *Include steps to test these changes*

## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### CODE REVIEW

File: prime-router/docs/getting-started/README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 - [Debugging](#debugging)
 - [Running the frontend](#running-the-frontend)
 - [Running the static site](#running-the-static-site)
+- [Push a sample message through the app](#push-a-sample-message-through-the-app)
```
Comment (arnejduranovic) [inline prime-router/docs/getting-started/README.md:17]:
```suggestion
- [Submit a sample report to ReportStream](#push-a-sample-message-through-the-app)
```

Comment (arnejduranovic) [reply prime-router/docs/getting-started/README.md:17]: This language better matches our preferred. Use this language elsewhere, like in the heading below.

File: prime-router/docs/getting-started/README.md [hunk 2]
```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
...
-See the [frontend React docs](../../../frontend-react/README.md) for more information.
+See the [frontend React docs](../../../frontend-react/README.md) for more information. Once you've done that and want to log in, __you need to use the same okta credentials you use when logging into the staging or production instances of the app__. 
+
+## Push a sample message through the app 
+
+1. If you haven't already, [download Postman](https://www.postman.com/downloads/) and install it. Import [the collections in the postman folder](./postman) into postman.
+
+2. If you haven't already, ensure the app is up and running. 
+
+3. Log into the LOCAL instance of the front end app. Do this by way of the front-end app and your CDC okta credentials.
```
Comment (arnejduranovic) [inline prime-router/docs/getting-started/README.md:171]:
```suggestion
3. Log into the LOCAL instance of the front end app. Do this by way of the front-end app and your STAGING CDC okta credentials.
```

File: prime-router/docs/getting-started/README.md [hunk 3]
```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
...
+
+4. In order to send a message, you are going to need add the "sender id" to the POST header. Click on "edit" on the "development" row.
```
Comment (arnejduranovic) [inline prime-router/docs/getting-started/README.md:173]:
```suggestion
4. In order to send a message, you are going to need to add the "sender id" to the POST header. Click on "edit" on the "development" row. 
```

File: prime-router/docs/getting-started/README.md [hunk 4]
```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
...
+
+2. If you haven't already, ensure the app is up and running. 
+
+3. Log into the LOCAL instance of the front end app. Do this by way of the front-end app and your CDC okta credentials.
+
+4. In order to send a message, you are going to need add the "sender id" to the POST header. Click on "edit" on the "development" row. 
+
+![report_stream_organizations.png](./img/report_stream_organizations.png)
+
+5. The data you seek is under "Organization Sender Settings". The client header is the sender. Concat the org name and the sender name should get you a valid value like so `client: development.dev-elims`
```
Comment (arnejduranovic) [inline prime-router/docs/getting-started/README.md:177]: Make this a sub bullet of 4? Or include it as part of 4?

File: prime-router/docs/getting-started/README.md [hunk 5]
```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
...
+
+4. In order to send a message, you are going to need add the "sender id" to the POST header. Click on "edit" on the "development" row. 
+
+![report_stream_organizations.png](./img/report_stream_organizations.png)
+
+5. The data you seek is under "Organization Sender Settings". The client header is the sender. Concat the org name and the sender name should get you a valid value like so `client: development.dev-elims` 
+
+![development_org_sender_settings.png](./img/development_org_sender_settings.png)
+
+6. In PostMan, open collection `ReportStream HL7 Demo (Local)` and select `POST Single HL7 Full ELR`. Then, under `headers` change the client value to `development.dev-elims` and press __Send__. If all went well you'll get an HTTP/201 response as shown below.
```
Comment (arnejduranovic) [inline prime-router/docs/getting-started/README.md:181]: postman

**Review by arnejduranovic — CHANGES_REQUESTED** [review]
Why do we have two postman collections here? Should we combine them into one?

File: prime-router/docs/getting-started/README.md (hunk 4 above)
Comment (david-navapbc) [reply prime-router/docs/getting-started/README.md:177]: wait - why? it's two actions on two pages. step (4) has you click on the "edit" button for the "development" row. that takes you to a new page. step (5) tells you what to do on the new page.

File: prime-router/docs/getting-started/README.md [hunk 6]
```diff
@@ -144,19 +148,40 @@ A `ctrl-c` will escape the running ReportStream instance.
...
 ## Running the frontend
 
-See the [frontend React docs](../../../frontend-react/README.md) for more information.
+See the [frontend React docs](../../../frontend-react/README.md) for more information. Once you've done that and want to log in, __you need to use the same okta credentials you use when logging into the staging or production instances of the app__. 
+
+## Push a sample message through the app 
+
+1. If you haven't already, [download Postman](https://www.postman.com/downloads/) and install it. Import [the collections in the postman folder](./postman) into postman.
+
+2. If you haven't already, ensure the app is up and running.
```
[the remark this answers is not in the document]
Comment (david-navapbc) [reply prime-router/docs/getting-started/README.md:169]: I'll leave it up to @arnejduranovic but my view is it's good to push new folks to fire up the front end app as well and do things end2end. It helped me at least seening things from the perspective of the UI.

File: prime-router/docs/getting-started/README.md (hunk 4 above)
Comment (david-navapbc) [reply prime-router/docs/getting-started/README.md:177]: oh! and re: the two postman collections @arnejduranovic 

One is the one you gave me that was also posted to slack a while back 

The other is the one I got from @JFisk42

Comment (JFisk42) [reply prime-router/docs/getting-started/README.md:177]: I can't even say if there are quality concerns about the postman collection that I linked. I do think it's good to have something like that available here. So, if there's any concern about adding a couple of large postman calls that might end up entirely or partially deprecated, a good idea might be to just upload a single postman call for this use case. That would be much more maintainable.

**Review by JFisk42 — APPROVED** [review]
I'm good with this so long as there aren't any objections around the postman collections.

File: prime-router/docs/getting-started/README.md (hunk 4 above)
Comment (arnejduranovic) [reply prime-router/docs/getting-started/README.md:177]: Changes look good, I just don't understand why we need two postman collections? `ReportStream.postman_collection.json` seems adequate to me?

Comment (david-navapbc) [reply prime-router/docs/getting-started/README.md:177]: ok - happy to use that one and remove the other one. I'll also update the screen shots where necessary

**Pull request merged**